app/parse-siti-payment-file/transform-header.js

```diff
@@ -0,0 +1,38 @@
+const { SFI, SFI_PILOT, LUMP_SUMS } = require('../schemes')
+
+const transformHeader = (headerData, scheme) => {
+  switch (scheme) {
+    case SFI:
+    case SFI_PILOT:
+      return transformSFIHeader(headerData)
+    case LUMP_SUMS:
+      return transformLumpSumsHeader(headerData)
+    default:
+      throw new Error('Unknown scheme')
+  }
+}
+
+const transformSFIHeader = (headerData) => ({
+  invoiceNumber: headerData[1],
+  paymentRequestNumber: parseInt(headerData[2]),
+  contractNumber: headerData[3],
+  frn: headerData[5],
+  currency: headerData[6],
+  value: parseFloat(headerData[7]),
+  deliveryBody: headerData[8],
+  schedule: headerData[11],
+  invoiceLines: []
+})
+
+const transformLumpSumsHeader = (headerData) => ({
+  invoiceNumber: headerData[1],
+  paymentRequestNumber: parseInt(headerData[2]),
+  contractNumber: headerData[3],
+  frn: headerData[4],
+  value: parseFloat(headerData[6]),
+  deliveryBody: headerData[7],
+  currency: headerData[8],
```

Comment:
would matching the order of the keys be more valuable than the order of the rows?

for e.g. I thought `lump sums` headers didn't have a `currency` looking at it for the first time